### PR TITLE
No weight decay for bias/norm/token/etc.

### PR DESCRIPTION
**Pull request by guarin**
## What has changed and why?

* Skip weight decay for bias/norm/token/etc.
* Reduce number of logged lr/wd values

This PR updates all models except LTDETR for which I'll create a follow-up PR.

## How has it been tested?

* Unit tests
* Manual tests

## Did you update [CHANGELOG.md](../CHANGELOG.md)?

- [x] Yes
- [ ] Not needed (internal change)

## Did you update the documentation?

- [ ] Yes
- [x] Not needed (internal change without effects for user)

### CODE REVIEW

File: src/lightly_train/_commands/train_task.py [hunk 1]
```diff
@@ -941,10 +941,11 @@ def _train_task_from_config(config: TrainTaskConfig) -> None:
                     task=config.task,
                 )
                 for group in optimizer.param_groups:
-                    train_log_dict[f"learning_rate/{group['name']}"] = group["lr"]
-                    train_log_dict[f"weight_decay/{group['name']}"] = group[
-                        "weight_decay"
-                    ]
+                    if group.get("log", True):
```
Comment (guarin) [inline src/lightly_train/_commands/train_task.py:947]: Allows skipping lr/wd logs for models with many optimizer param groups

**Comment by guarin** [comment]
> I was wondering whether we could put some of the helper into LightlySSL?

The weight decay / no weight decay helper function is in LightlySSL although not with all the functionality. We could definitely also move it there.

**Comment by guarin** [comment]
@codex review
